README.md

```diff
@@ -148,7 +149,7 @@ To run the project, execute the following command:
 > Describe 1 - 3 features you will add to the project.
 
 - [ ] **[Update animals table]**
-- [ ] **[Add more Querries]**
+- [ ] **[Add more Queries]**
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
```

Comment:
- [ ] The link to your LICENSE is broken and redirects to GitHub's error page. Kindly fix the LICENSE. You can add a valid LICENSE by following the instructions in this [article](https://choosealicense.com/licenses/mit/). 👍 😎

Reply:
Hey @Gambit142 Thank you for the observation. I have added the license to my project